dbms/src/Functions/CMakeLists.txt

```diff
@@ -69,3 +69,8 @@ if (USE_HYPERSCAN)
     target_link_libraries (clickhouse_functions PRIVATE ${HYPERSCAN_LIBRARY})
     target_include_directories (clickhouse_functions SYSTEM PRIVATE ${HYPERSCAN_INCLUDE_DIR})
 endif ()
+
+if (HAVE_AVX2)
```

Comment:
Did you define `HAVE_AVX2` somewhere?

Reply:
It is generated by cmake. It still have some trouble here.